### PR TITLE
Rewriting tests

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

File: complete/src/test/java/hello/HelloWorldConfigurationTests.java [hunk 1]
```diff
@@ -19,50 +19,53 @@
...
 	@Value("${local.management.port}")
 	private int mgt;
 
+	@Autowired
+	private TestRestTemplate testRestTemplate;
+
 	@Test
-	public void testGreeting() throws Exception {
+	public void should_return_200_when_sending_request_to_controller() throws Exception {
 		@SuppressWarnings("rawtypes")
```
Comment (gregturn) [inline complete/src/test/java/hello/HelloWorldConfigurationTests.java]: I haven't looked at other Spring projects yet on BDD, but I find **shouldReturn200WhenSendingRequestToController** a bit less jarring.

File: complete/src/test/java/hello/HelloWorldConfigurationTests.java [hunk 2]
```diff
@@ -19,50 +19,53 @@
...
 				"http://localhost:" + this.port + "/hello-world", Map.class);
-		assertEquals(HttpStatus.OK, entity.getStatusCode());
+
+		then(entity.getStatusCode()).isEqualTo(HttpStatus.OK);
 	}
 
 	@Test
-	public void testInfo() throws Exception {
+	public void should_return_200_when_sending_request_to_management_endpoint() throws Exception {
 		@SuppressWarnings("rawtypes")
```
Comment (gregturn) [inline complete/src/test/java/hello/HelloWorldConfigurationTests.java]: Same here.

File: complete/src/test/java/hello/HelloWorldConfigurationTests.java (hunk 1 above)
Comment (dsyer) [inline complete/src/test/java/hello/HelloWorldConfigurationTests.java]: Me too.

Comment (marcingrzejszczak) [inline complete/src/test/java/hello/HelloWorldConfigurationTests.java]: You don't know what's good for you... :> I will unify it then even though personally I think it's better to read it with underscores.

**Pull request merged**